### PR TITLE
Add ListGroups command

### PR DESCRIPTION
**Pull request by pimotte**
Option: -d/--detailed to display group members

As discussed in #58, this might be useful. 

Happy to hear any and all feedback:)

### CODE REVIEW

File: src/main/java/org/asamk/signal/Main.java [hunk 1]
```diff
@@ -492,6 +492,22 @@ public void handle(Signal.MessageReceived s) {
...
+                    }
+                    if (!m.isRegistered()) {
+                        System.err.println("User is not registered.");
+                        return 1;
+                    }
+    
+                    List<GroupInfo> groups = m.getGroups();
+    
+                    for (GroupInfo group : groups) {
+                        printGroup(group, ns.getBoolean("detailed"));
```
Comment (AsamK) [inline src/main/java/org/asamk/signal/Main.java]: Please create a variable `boolean detailed` outside the for loop for this.

File: src/main/java/org/asamk/signal/Manager.java [hunk 2]
```diff
@@ -580,6 +580,10 @@ private GroupInfo getGroupForSending(byte[] groupId) throws GroupNotFoundExcepti
         }
         throw new NotAGroupMemberException(groupId, g.name);
     }
+    
+    public List<GroupInfo> getGroups() {
+    	return groupStore.getGroups();
```
Comment (AsamK) [inline src/main/java/org/asamk/signal/Manager.java]: Fix indentation, tab -> spaces

File: src/main/java/org/asamk/signal/Main.java [hunk 3]
```diff
@@ -622,6 +638,17 @@ private static void printIdentityFingerprint(Manager m, String theirUsername, Js
         System.out.println(String.format("%s: %s Added: %s Fingerprint: %s Safety Number: %s", theirUsername,
                 theirId.trustLevel, theirId.added, Hex.toStringCondensed(theirId.getFingerprint()), digits));
     }
+    
+    private static void printGroup(GroupInfo group, boolean detailed) {
+        System.out.println(String.format("Group id: %s\n Group name: %s \n active: %s",
```
Comment (AsamK) [inline src/main/java/org/asamk/signal/Main.java]: I'd prefer having each group in a single line, e.g.:
"Id: %s Name: \"%s\" Active: %s Members: ..."

**Comment by pimotte** [comment]
Fixed all of those:)

**Pull request merged**